libs/platform/src/lib/smart-filter-bar/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+    "extends": "../../../../../tsconfig.base.json",
+    "files": [],
+    "include": [],
+    "references": [
+        {
+            "path": "./tsconfig.lib.json"
+        },
+        {
+            "path": "./tsconfig.lib.prod.json"
+        },
+        {
+            "path": "./tsconfig.spec.json"
+        }
+    ],
+    "compilerOptions": {
+        "forceConsistentCasingInFileNames": true,
+        "strict": false,
+        "noImplicitReturns": false,
+        "noFallthroughCasesInSwitch": true
+    },
+    "angularCompilerOptions": {
+        "strictInjectionParameters": false,
+        "strictTemplates": false
+    }
```

Comment:
can both of these be enabled?

Reply:
@dmitry-stepanenko since smart filter bar uses other components where strict mode is not enabled, there's errors during build with incompatible data types, which are actually compatible. So those components should be strict first.